css/style.css

```diff
@@ -0,0 +1,77 @@
+@import "https://fonts.googleapis.com/css?family=Crete Round|Inter";
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'crete round', sans-serif;
+}
+
+body {
+  width: 100%;
+  height: auto;
+  position: relative;
+}
+
+h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  align-self: center;
+  justify-content: center;
+  margin: 6px 24px;
+  font-family: 'crete round', sans-serif;
```

Comment:
- [ ] Kindly review the fonts provided on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=5eryAHFhbT9gK0l9-0) to match your current design with that on the figma design. Yours seems a little bit bold. the exact width was provided n the link I attached 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219271155-01ddcbbd-1b77-4c4d-9a63-456413e548e1.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/219271211-2e3bcca2-44d4-479f-8735-9ba927d6eca9.png)